qrand/platforms/qsharp/backend.py

```diff
@@ -20,22 +20,37 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 
+from typing import Optional
+
 from ..backend import QuantumBackend
 
 
 ###############################################################################
 ## QSHARP BACKEND (DECORATOR)
 ###############################################################################
 class QsharpBackend(QuantumBackend):
-    def __init__(self) -> None:
-        self.ERROR_MSG = f"{self.__class__.__name__}"  # TODO
-        raise NotImplementedError(self.ERROR_MSG)
+    def __init__(
+        self,
+        resource_id: Optional[str] = None,
+        target_id: Optional[str] = None,
+    ) -> None:
+        self.resource_id = resource_id
+        self.target_id = target_id
 
     ############################### PUBLIC API ###############################
     @property
     def max_measurements(self) -> int:
-        raise NotImplementedError(self.ERROR_MSG)
+        return 1048576
```

Comment:
Was this from a particular partner backend?

Reply:
No, but it confirms to Qiskit's simulator.

Reply:
This is roughly the maximum that `qiskit` allows (2^19 < 2^13*75 < 2^20). It was added for consistency and more as a default value. I will need to think about how to refactor, but I will take care of it myself. Any further thoughts @crazy4pi314?